docs/news/index.qmd

```diff
@@ -9,6 +9,47 @@ format:
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2024.09.1
+
+**"Cranberry Hibiscus"**
+
+>Date: 2024-11-04
+
+### New
+
+#### RStudio
+
+#### Posit Workbench
+
+- Add support for conditional regex replacement strings in proxied auth (rstudio-pro#6753)
+
+### Fixed
+
+#### RStudio
+
+- Fixed an issue where the R session could crash on connect while generating console output (#15330)
+- Fixed Visual Editor losing raw HTML code blocks (#15189)
+- Fixed Visual Editor losing raw LaTeX code blocks (#15253)
+- Fixed opening and saving files on UNC paths (Windows Desktop, #15280)
+- Fixed package builds failing when set to generate documents on build (#15236)
```

Comment:
I think @AshleyHenry15 might say these lines should be terminated by periods, but I can't remember 100% and Confluence's search isn't going to help me find out whether we have this written down.

Reply:
Done!